assets/src/edit-story/app/media/utils/useUploadVideoFrame.js

```diff
@@ -56,15 +60,35 @@ function useUploadVideoFrame({ updateMediaElement }) {
         featured_media: posterId,
         post: storyId,
       });
+
+      // Preload the full image in the browser to stop jumping around.
+      // It's an asynchronous operation, but there's no need to await it.
+      preloadImage(poster);
```

Comment:
Awaiting here was by design. It you dont await the photo to download, it jumps in the UI.

Reply:
Sounds good. I brought the `await` back.